kamu-cli/src/explore/api_server.rs

```diff
@@ -81,3 +139,157 @@ async fn graphql_playground() -> impl axum::response::IntoResponse {
         async_graphql::http::GraphQLPlaygroundConfig::new("/graphql"),
     ))
 }
+
+async fn dataset_refs_handler(
```

Comment:
What do you think about moving all this out of `kamu-cli` module into `kamu-adapter-http`? In the spirit of hexagonal architecture we will then have a self-contained module implementing simple & smart HTTP-based ODF protocols which can be included here into overall routing table alongside the `kamu-adapter-gql`.

Reply:
Added `kamu-adapter-http`, where we will host the protocol implementation